packages/examples/src/form/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Document,
+  Page,
+  View,
+  Text,
+  Form,
+  Checkbox,
+  FormField,
+  TextInput,
+  Picker,
+  FormPushButton,
+  FormList,
+} from '@react-pdf/renderer';
+
+const FormPdf = () => (
+  <Document>
+    <Page>
+      <View
+        style={{
+          backgroundColor: 'rgba(182,28,28,0.62)',
+          width: '30%',
+          height: '100%',
+        }}
+      >
+        <Form>
+          <FormField name="user-info" style={{ flexDirection: 'column' }}>
+            <Text>TextInput</Text>
+            <TextInput
+              name="username"
+              value="foo"
+              align="center"
+              style={{ height: '50px' }}
+            />
+
+            {/* Nested works as well */}
+            <View>
+              <Text>TextInput</Text>
+              <TextInput
+                name="password"
+                value="bar"
+                align="center"
+                style={{ height: '50px' }}
+                password
+              />
+            </View>
+
+            <Text>Checkbox</Text>
+            {/* TODO: how to check automatically? */}
+            <Checkbox name="checkbox" style={{ height: '20px' }} />
```

Comment:
- [ ] example: how to set `checked`?

Reply:
Thank you! This works like a charm. Thank you for sharing the code! I'd have no idea how to do this myself currently.

Reply:
Updated the code for an appearance with paths.
I don't own a Mac. Would be thankful if you could try this one and give feedback on it.

Reply:
> Updated the code for an appearance with paths. I don't own a Mac. Would be thankful if you could try this one and give feedback on it.

👋  I think we are getting there, but not yet.

Web:
<img width="327" alt="image" src="https://github.com/user-attachments/assets/d2b5c84f-7b2d-4b2d-bcf8-194980cb1c0e">

macOS:

<img width="238" alt="image" src="https://github.com/user-attachments/assets/5f4204b7-c661-49a7-b25d-d0b0a599ad0a">

Reply:
While tinkering around I noticed my mistake on the first try. I wanted to use the [Zapf Dingbats](https://en.wikipedia.org/wiki/Zapf_Dingbats)-Font for the marks, but put neither a ref nor the resource into the code. I tried to fix that one, because it should be the cleaner one of the two.

By removing `NeedAppearances` from pdfkit I could recreate an error state with the first one and at least could verify that my fix worked on chrome on Windows. From the recreation I would conclude that the apple standard PDF viewer doesn't support `NeedAppearances` which would make it even more important to work right. Thanks for trying out my solutions.

Sidenote: I had the idea to put an option for xmarks in. So that it shows checkmarks default, but the user can decide to change it. I also fixed two points where you accidentally compared a string with a boolean via triple equals which will always result in a false expression.

Reply:
Hi @PhilippBloss, thanks for your help. Without your support, this would not work! 💯 

It works now on Web (Screenshot 1) and macOS (Screenshot 2):

### Web
<img width="263" alt="image" src="https://github.com/user-attachments/assets/f550742f-0448-4aff-b58b-df8530086986">

### macOS
<img width="238" alt="image" src="https://github.com/user-attachments/assets/074f3b78-ee57-4121-b851-3c8c1b737159">